tests/command_line/test_indexed_as_integrated.py

```diff
@@ -0,0 +1,48 @@
+import procrunner
+
+import iotbx.merging_statistics
+
+
+def test_indexed_as_integrated(dials_data, tmp_path):
+    data_dir = dials_data("insulin_processed", pathlib=True)
+    refl = data_dir / "indexed.refl"
+    expt = data_dir / "indexed.expt"
+
+    command = [
+        "dials.indexed_as_integrated",
+        refl,
+        expt,
+        "output.reflections=output.refl",
+    ]
+    result = procrunner.run(command, working_directory=tmp_path)
```

Comment:
I would probably prefer to call the command line `run` method directly, rather than with an additional indirection layer via `procrunner` that complicates any tracebacks or warnings etc, but that isn't critical to merging this PR.